programs/survival/bee_healing.sc

```diff
@@ -0,0 +1,15 @@
+// Bees have an annoying habit of drowning themselves, so yeah
+__config()->{'stay_loaded' -> true, 'scope' -> 'global'};
```

Comment:
again - stay loaded is true by default - no need to specify that.

Reply:
I wrote the scripts when that wasn't case, I guess I just forgot to change it. I've removed it now tho